ci/do_ci.sh

```diff
@@ -47,7 +47,8 @@ function do_unit_test_coverage() {
 
 function do_integration_test_coverage() {
     export TEST_TARGETS="//test:python_test"
-    export COVERAGE_THRESHOLD=78.6
+    #TODO(#564): Revert this to 78.6
+    export COVERAGE_THRESHOLD=75
```

Comment:
This is fine, but looking at this, maybe we should leave a note here that the comparison is limited to one tenths of a percent (as that is the precision at which is gets determined by the coverage flow).

Reply:
That makes sense thanks.